partner-api.yaml

```diff
@@ -531,51 +531,80 @@ components:
         _links:
           $ref: '#/components/schemas/PartnerLinks'
 
+    WritablePartner:
+      type: object
+      properties:
+        titelFunktion:
+          type: string
+        anrede:
+          type: string
+          enum:
+            - HERR
+            - FRAU
+          example: FRAU
+        vorname:
+          type: string
+          description: "Dieses Feld ist nur für den Typ 'PERSON' gefüllt."
+        nachname:
+          type: string
+          description: "Dieses Feld ist nur für den Typ 'PERSON' gefüllt."
+        geburtsdatum:
+          type: string
+          format: date
+          description: ISO-8601 Calender extended(YYYY-MM-DD) format.
+          example:
+            1980-12-11
+        telefonnummer:
+          type: string
+        mobilnummer:
+          type: string
+        aufsichtsbehoerde:
+          type: string
+        registrierungsnummer:
+          type: string
+        anschrift:
+          $ref: '#/components/schemas/Anschrift'
+        bankverbindung:
+          $ref: '#/components/schemas/Bankverbindung'
+        email:
+          type: string
+          format: email
+          example:
+            john.doe@example.org
+        externePartnerId:
+          type: string
+          description: 'Eine beliebige, extern erzeugte Id. Z.B. SAP oder CRM Nummer.'
+        faxnummer:
+          type: string
+        firmenname:
+          type: string
+        firmennameZusatz:
+          type: string
+          description: Zusatz zur Firma (z.B. Abteilung)
+        webseite:
+          type: string
+          format: uri
+        name:
+          type: string
+          description: "Dieses Feld ist nur für den Typ 'ORGANISATION' gefüllt."
+        typ:
+          $ref: '#/components/schemas/Typ'
+        avatar:
+          type: string
+          format: uri
+        gesperrt:
+          type: boolean
+          default: false
+          description: 'true, wenn der Partner oder ein übergeordneter Partner gesperrt ist.'
+        kreditsachbearbeiter:
+          type: boolean
+
     Partner:
       title: Partner
       allOf:
         - $ref: '#/components/schemas/Plakette'
+        - $ref: '#/components/schemas/WritablePartner'
```

Comment:
hast du versucht das zu generieren? Ich könnte mir vorstellen, dass es da probleme gibt, weil es Dinge wie Name etc. ja auch in der Plakette gibt, oder?

Reply:
Hast natürlich recht. Ich versuche das auch mal zu generieren :+1: